ansible/playbooks/rook.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: repository
+  gather_facts: true
+  tasks: []
+
+- hosts: kubernetes_master
```

Comment:
Will it be executed multiple times in HA installation?

Reply:
Shouldn't be.

Reply:
Limited to only first kubernetes_master.